src/vmdeps.txt

```diff
@@ -3,7 +3,11 @@
 # rpm-ostree runs (when run unprivileged) and create-disk.sh.
 
 # bare essentials
-bash vim-minimal coreutils util-linux procps-ng kmod kernel-modules
+bash vim-minimal coreutils util-linux procps-ng kmod
+
+# Pin to 5.11 until https://github.com/openshift/os/issues/646 is root caused
+# (May be related to https://bugzilla.redhat.com/show_bug.cgi?id=2008401)
+kernel-modules-5.11.12-300.fc34
```

Comment:
A more explicit approach is something like https://github.com/coreos/coreos-assembler/pull/2014, but instead of using Koji URLs, since it's in the repos, we can do e.g.

```sh
frozendeps=$(echo kernel{,-core,-modules}-5.13.19-200.fc34)
```